src/modules/profile/components/Menu.scss

```diff
@@ -0,0 +1,50 @@
+@import 'src/styles/settings/general';
+
+.menuContainer {
+    position: absolute;
+    margin-top: -5rem;
+    right: 0rem; 
+    background-color: $brand-blue;
+    z-index: -1;
+    transition: 1s margin-top ease;
+    -webkit-transition: 1s margin-top ease;
```

Comment:
What do you need -webkit- prefixes for in this file?  Isn't this polyfilled, and if not, does this then work in other browsers?